xbmc/cores/VideoPlayer/VideoPlayerAudio.h

```diff
@@ -115,9 +115,7 @@ class CVideoPlayerAudio : public CThread, public IDVDStreamPlayerAudio
   SInfo            m_info;
 
   bool m_displayReset = false;
-  unsigned int m_disconAdjustTimeMs = 10; // maximum sync-off before adjusting
+  unsigned int m_disconAdjustTimeMs = 50; // maximum sync-off before adjusting
```

Comment:
Make it PT depending? For "non passthrough" audio, periods might be smaller and we could actually optimize towards even lower values.

Reply:
It is not necessary because of the way the corrections are made: a frame is advanced or repeated (this is steps of ~40 ms most of the times). It is not possible better adjust in "DISCON" mode anyway. And this is no used at all in resample mode (Sync playback to display enabled)

Reply:
It is not very intuitive because the code is in the audio part but the adjustments are made by touching the video clock...

Reply:
Okay. My reasoning was that with normal PCM audio, resampled or not, one could skip / drop / replace with zero certain samples even more fine granular than 50 ms. Which is most likely why we had 10 ms here in the first place. For PT 10 ms might make sense for DTS which is 10 ms length packages, but not much sensor for AC3, etc.

If only video is adjusted ... then anyways make not too much sensen then.